jsonpb/jsonpb.go

```diff
@@ -77,6 +77,11 @@ type Marshaler struct {
 	// Whether to use the original (.proto) name for fields.
 	OrigName bool
 
+	// Skips qouting uint64, int64, and fixed64 types as strings. Default
+	// behavior is to quote per protobuf spec
+	// Ref: https://developers.google.com/protocol-buffers/docs/proto3#json
+	SkipQouteInt64 bool
```

Comment:
`s/Qoute/Quote/`

Reply:
Welp, that is super embarrassing... :blush: 

fixed all the spelling and branch name in: https://github.com/golang/protobuf/pull/916